lib/cucumber/blendle/steps/rest_steps.rb

```diff
@@ -133,3 +123,25 @@
     assert last_response.headers[header], "missing header: #{header}"
   end
 end
+
+# * Then the response at "_embedded/b:subscription-product" should include:
```

Comment:
@broeken just a little nit: this is a public repository, so it makes sense to use generic examples, whereas this is a Blendle-specific example.